### ISSUE TITLE
Button Dialog Issue

### ISSUE THREAD

**Issue by eparascho** [algogr/HourglassQml#2]
Pressing the Dialog button does not display a box other than the original state. It looks like the application does not support this feature.
I tried running the application from both Windows and Linux OS.
The result was the same.

https://user-images.githubusercontent.com/56228687/105353263-b2d0a500-5bf7-11eb-92aa-ba8e620de58b.png

https://user-images.githubusercontent.com/56228687/105353288-b8c68600-5bf7-11eb-8037-73c140a4322e.png

**Comment by algogr** [comment]
Yes this is true. It does nothing an it is a bug. Would you like helping to resolve?

**Comment by eparascho** [comment]
I'm so sorry but I will not be able to participate in the bug fix due to lack of time and experience.
